benchmarking/config.ini

```diff
@@ -17,6 +17,12 @@ USERS = users.csv
 CHATS = chats.csv
 WRITE_OUTPUT = ResponseEntry.csv
 PROCESS_OUTPUT = latencies.csv
+ERROR = errors.txt
+
+[Load Test]
+WARM_UP_MIN_QPS = 50
+WARM_UP_MAX_QPS = 150
+WARM_UP_DURATION = 2
```

Comment:
What's the unit here?
Let's call it WARM_UP_DURATION_MINUTES instead for clarity.

Reply:
Changed.